src/Common/Core/Impl/Shell/IColorService.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Common.Core.Shell {
+    public interface IColorService {
+        bool IsDarkTheme { get; }
```

Comment:
I still think we should move it to the `IApplicationConstants` and have a property that returns enum. Something like `ApplicationTheme`.